mon-pix/tests/acceptance/start-campaigns-workflow-for-pole-emploi_test.js

```diff
@@ -0,0 +1,225 @@
+/* eslint ember/no-classic-classes: 0 */
+
+import { beforeEach, describe, it } from 'mocha';
+import { expect } from 'chai';
+
+import { fillIn, currentURL, find, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-mocha';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { Response } from 'ember-cli-mirage';
+
+import { clickByLabel } from '../helpers/click-by-label';
+import sinon from 'sinon';
+import Service from '@ember/service';
+
+import { authenticateByEmail } from '../helpers/authentication';
+import { currentSession } from 'ember-simple-auth/test-support';
+import setupIntl from '../helpers/setup-intl';
+import { t } from 'ember-intl/test-support';
+
+describe('Acceptance | Campaigns | Start Campaigns workflow | Pôle Emploi', function () {
```

Comment:
Idem ici, je pense qu'idéalement on voudrait plutôt un "Campaign with an identity provider workflow" et pas quelque chose qui soit spécifique à un identity provider, à moins qu'il y ait dans le workflow quelque chose de spécifique à cet idp.

Reply:
c'est compliqué de faire générique en acceptance, surtout qu'ici on a des `expect(currentURL()).to.equal('/connexion-pole-emploi');` ... je peux rendre générique les intitulés de test mais je suis pas sûre que ça apporte plus que ça n'apporte du flou 🤔